integration/e2ecortex/client.go

```diff
@@ -391,6 +389,32 @@ type userConfig struct {
 	AlertmanagerConfig string            `yaml:"alertmanager_config"`
 }
 
+// GetAlertmanagerRingStatusPage gets the status page of alertmanager.
+func (c *Client) GetAlertmanagerStatusPage(ctx context.Context) ([]byte, error) {
+	return c.getRawPage(ctx, "/multitenant_alertmanager/status")
```

Comment:
The integration tests are failing because of this:
```
    alertmanager_test.go:457: 
        	Error Trace:	alertmanager_test.go:457
        	Error:      	Received unexpected error:
        	            	Get "/multitenant_alertmanager/status": unsupported protocol scheme ""
        	Test:       	TestAlertmanagerSharding/RF_=_3
```

Reply:
You can store `alertmanagerAddress` (taken in input by `NewClient()`) in the `Client` struct and then setting the URL here as:

```suggestion
	return c.getRawPage(ctx, "http://" + c.alertmanagerAddress + "/multitenant_alertmanager/status")
```